src/jack.c

```diff
@@ -388,9 +388,11 @@ jalv_backend_init(Jalv* jalv)
 void
 jalv_backend_close(Jalv* jalv)
 {
-	jack_client_close(jalv->backend->client);
-	free(jalv->backend);
-	jalv->backend = NULL;
+	if (!jalv->backend) {
```

Comment:
@drobilla I think you have missed to remove the `!` here

Reply:
Should I create a new PR for this change or could you fix it directly?

Reply:
Oof, oops.  That's okay, I'll fix it